app/src/main/java/com/framgia/forder/screen/ordershop/ItemChildOrderViewModel.java

```diff
@@ -1,8 +1,45 @@
 package com.framgia.forder.screen.ordershop;
 
+import com.framgia.forder.data.model.OrderDetail;
+
 /**
  * Created by levutantuan on 6/1/17.
  */
 
+@SuppressWarnings({ "SameReturnValue", "DefaultFileTemplate" })
 public class ItemChildOrderViewModel {
+    private final OrderDetail mOrderDetail;
+
+    ItemChildOrderViewModel(OrderDetail orderDetail) {
+        mOrderDetail = orderDetail;
+    }
+
+    public String getStatus() {
+        if (mOrderDetail != null) {
+            mOrderDetail.getStatus();
+        }
+        return "";
+    }
+
+    public int getQuantity() {
+        if (mOrderDetail != null) {
+            mOrderDetail.getQuantity();
+        }
+        return 0;
+    }
+
+    public String getPrice() {
+        if (mOrderDetail != null) {
+            mOrderDetail.getFormatTotalPrice();
+        }
+        return "";
+    }
+
+    public String getNotes() {
+        if (mOrderDetail != null) {
+            mOrderDetail.getNotes();
```

Comment:
ERROR: [STYLE][RV_RETURN_VALUE_IGNORED_NO_SIDE_EFFECT] Return value of com.framgia.forder.data.model.OrderDetail.getNotes() ignored, but method has no side effect
http://findbugs.sourceforge.net/bugDescriptions.html#RV_RETURN_VALUE_IGNORED_NO_SIDE_EFFECT